app/views/catalog/_universal_viewer_issue.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if resource.class.can_have_manifests? && resource.try(:member_ids).present? && !resource.decorate.decorated_coins.map(&:decorated_file_sets).flatten.empty?  %>
```

Comment:
Rather than using the decorator can we do something like `Wayfinder.for(resource).coin_file_sets.empty?`?